resourcepacks/HLTweaker Lines/assets/hltweaker/lang/en_us.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Translations for debugging purposes, they are not shown unless you enable it in config https://github.com/yzl210/HarvestLevelTweaker?tab=readme-ov-file#localization-and-custom-vanilla-level-names